tools/utility/dockerise/run.py

```diff
@@ -17,7 +17,11 @@
 
 def _is_docker():
     path = "/proc/self/cgroup"
-    return os.path.exists("/.dockerenv") or os.path.isfile(path) and any("docker" in line for line in open(path))
+    return (
+        os.path.exists("/.dockerenv")
+        or os.path.isfile(path)
```

Comment:
Can you please add brackets around this condition?

reading this, it's not clear to me when:
- the first condition is false
- the second condition is true

what happens? 

If operator precedence of `or` and `and` in python is equivalent, then the third condition never matters - the `and` requires the second condition to be true, but if the second condition is true and the precedence is the same, then the `or` is true, and the whole statement is true, because short-circuit.
Please just make this more clear, either way.